### PR TITLE
Turn 3d

### PR DESCRIPTION
**Pull request by fsandel**
First stable version of 3d version of our cub.

Colission is added,
Fisheye is ok,
player movement got restructured

### CODE REVIEW

File: Makefile
Comment (philippschwarzmueller) [inline Makefile:1]: remove header pls

**Review by philippschwarzmueller — CHANGES_REQUESTED** [review]
pls refer to my comments

File: testmap.txt
[the remark this answers is not in the document]
Comment (fsandel) [reply testmap.txt:1]: removed file

**Review by philippschwarzmueller — APPROVED** [review]
lgtm

**Pull request merged**